### PR TITLE
UIREQ-962 added request.requestDate token

### PR DESCRIPTION
**Pull request by rgupta-nla**
## Purpose
Update staff slips to have the request date

## Approach
add new token for request date

## Refs
https://issues.folio.org/browse/UIREQ-962

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by artem-blazhko — COMMENTED** [review]
Could you add appropriate changes to CHANGELOG.md file?

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@
 * Cover `src/components/Loading/Loading.js` file by RTL/Jest tests. Refs UIREQ-885.
 * Leverage cookie-based authentication in all API requests. Refs UIREQ-861.
 * Update `circulation` okapi interface to `14.0` version. Refs UIREQ-954.
+* Added requestDate token. Refs CIRC-1784.
```
Comment (Dmitriy-Litvinenko) [inline CHANGELOG.md:24]: Hello how i understood CIRC (mod-circulation) this is BE Story, you should have corresponding FE Story and provide your changes in scope of FE Story in current case UIREQ (ui-requests)

Comment (rgupta-nla) [reply CHANGELOG.md:24]: I do not have separate FE story. Also, the FE change is spread across three different repositories. ui-circulation, ui-request and ui-checkin.
I am just adding a new token for requestDate.

**Comment by rgupta-nla** [comment]
I have now created separate UIREQ jira ticket and updated this PR title and description.

File: CHANGELOG.md (hunk 1 above)
Comment (Dmitriy-Litvinenko) [inline CHANGELOG.md:24]: Could you please update CHANGELOG.md how i understood we should change from  `* Added requestDate token. Refs CIRC-1784.` to `* Added requestDate token. Refs UIREQ-962.`

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Review by zburke — APPROVED** [review]

**Pull request merged**